src/Toolkit.Forms/OverviewMap/OverviewMap.cs

```diff
@@ -0,0 +1,226 @@
+                                                 xmlns:x=""http://schemas.microsoft.com/winfx/2009/xaml""
+                                                 xmlns:esri=""clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms""
+                                                 xmlns:internal=""clr-namespace:Esri.ArcGISRuntime.Toolkit.Xamarin.Forms.Internal;assembly=Esri.ArcGISRuntime.Toolkit.Xamarin.Forms"">
+                                    <Frame Padding=""1"" WidthRequest=""100"" HeightRequest=""100"" CornerRadius=""0"" HasShadow=""False"" BorderColor=""Black"" BackgroundColor=""White"">
+                                    <Grid>
+                                        <Grid.Resources>
+                                            <internal:LoadStatusToVisibilityConverter x:Key=""LoadStatusToVisibilityConverter"" />
+                                        </Grid.Resources>
+                                        <ActivityIndicator IsRunning=""{Binding Source={x:Reference PART_MapView}, Path=Map.LoadStatus, Converter={StaticResource LoadStatusToVisibilityConverter}, ConverterParameter='Loading'}"" />
+                                        <Label TextColor=""Black"" Text=""Map failed to load. Did you forget an API key?"" IsVisible=""{Binding Source={x:Reference PART_MapView}, Path=Map.LoadStatus, Converter={StaticResource LoadStatusToVisibilityConverter}, ConverterParameter='FailedToLoad'}""  />
+                                        <esri:MapView x:Name=""PART_MapView"" IsAttributionTextVisible=""False"" IsVisible=""{Binding Source={x:Reference PART_MapView}, Path=Map.LoadStatus, Converter={StaticResource LoadStatusToVisibilityConverter}, ConverterParameter='Loaded'}"" />
+                                    </Grid>
+                                    </Frame>
+                                </ControlTemplate>";
+            DefaultControlTemplate = Extensions.LoadFromXaml(new ControlTemplate(), template);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OverviewMap"/> class.
+        /// </summary>
+        public OverviewMap()
+        {
+            ControlTemplate = DefaultControlTemplate;
+            AreaSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.Null, System.Drawing.Color.Transparent, new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.Red, 1));
+            PointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Cross, System.Drawing.Color.Red, 16);
+            Map = new Map(BasemapStyle.ArcGISTopographic);
+        }
+
+        /// <summary>
+        /// <inheritdoc />
+        /// </summary>
+        protected override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+
+            _overviewMapView = GetTemplateChild("PART_MapView") as MapView;
+
+            if (_overviewMapView != null)
+            {
+                _overviewMapView.Map = Map;
+
+                _controller?.Dispose();
```

Comment:
Do this _before `_overviewMapView` is assigned in line 74.
Ie before line 74, you clean up from the old round (also important to unhook the old map, since it can only be used on one mapview):
```cs
if (_overviewMapView != null)
{
    _overviewMapView.Map = null;
    _controller?.Dispose();
}
```